scripts/plot_em_vs_bran_v1.cpp

```diff
@@ -14,7 +14,7 @@ namespace axisConfig = axis::singleNeutron;
 /**
  * @brief plot EM module truth vs BRAN truth
  * this version is simpler and plots with an event loop, using TTreeReader
- * run with, for example: root -b -q -l plot_em_vs_bran_v1.cpp
+ * run with, for example: root -b -q -l scripts/plot_em_vs_bran_v1.cpp
```

Comment:
This is correct if you're executing ROOT from the _root_ directory of the repo, but I have been assuming that we execute `root ...` from the scripts directory. Either is fine, but I've changed the paths in our macros such that the scripts can be executed from the scripts directory, for consistency.